setup.sh

```diff
@@ -55,8 +57,9 @@ elif [[ -z "${OPS_PROJECT}" ]]; then
     exit 1
 fi
 
-## Initialize Variables ##
-export REGION="us-central1"
+if [[ -z "${REGION}" ]]; then
```

Comment:
We might be able to wrap this into the change above 🙂 

[[Relevant Stackoverflow post](https://stackoverflow.com/a/2013589)]

Reply:
good idea